pytamp_moveit_bridge/ObjectState[] objects
